### ISSUE TITLE
Notifier double close

### ISSUE THREAD

**Issue by vmihailenco** [airbrake/gobrake#22]
```
panic: close of closed channel [recovered] 
panic: close of closed channel [recovered] 
panic: close of closed channel

goroutine 15 [running]: 
panic(0x369440, 0xc82043c000) 
/usr/local/go/src/runtime/panic.go:481 +0x3e6 
testing.tRunner.func1(0xc820094090) 
/usr/local/go/src/testing/testing.go:467 +0x192 
panic(0x369440, 0xc82043c000) 
/usr/local/go/src/runtime/panic.go:443 +0x4e9 
gopkg.in/airbrake/gobrake%2ev2.(*Notifier).NotifyOnPanic(0xc8200180e0)
```

**Comment by jvalentine** [comment]
Thanks for creating this issue, it's myself who got the error.

**Comment by vmihailenco** [comment]
@jvalentine hopefully fixed in v2.0.7. Thanks for the report.
